api/model/src/test/java/org/projectnessie/error/TestNessieError.java

```diff
@@ -67,8 +67,7 @@ static Stream<Arguments> conflictDeserialization() {
                 .put("message", "msg")
                 .put("someNewField", "blah")
                 .set("key", ckJson),
-            conflict(UNKNOWN, ContentKey.of("foo"), "msg")),
-        arguments(objectNode().put("message", "msg"), conflict(null, null, "msg")));
```

Comment:
Would it be worth keeping this test case to validate the logic on line 111 in `[Conflict.java](https://github.com/projectnessie/nessie/pull/7667/files#diff-ef0143fc6d8dab3c14a453ada3e4ae98263f7211578ec223921369753da19260)`? Perhaps we could change the expected value to use `UNKNOWN`? WDYT?

Reply:
I tried, but it doesn't work:

```
com.fasterxml.jackson.databind.exc.ValueInstantiationException: Cannot construct instance of `org.projectnessie.model.ImmutableConflict`, problem: Cannot build Conflict, some of required attributes are not set [conflictType]
 at [Source: UNKNOWN; byte offset: #UNKNOWN]
```

Reply:
Hmm wait. This is suspicious. It should work. I'll investigate.